src/requests.rs

```diff
@@ -34,7 +34,7 @@ mod reqwest_support {
 
             let mut response = self.get(url).send()?;
             let mut res_str = String::new();
-            response.read_to_string(&mut res_str).map_err(reqwest::HyperError::from)?;
```

Comment:
error handling has turned into a panic, definitely no good

Reply:
Right.. I should probably update the `Error` type of this `impl` to include io errors. Not sure how to do that though... (do I have to use something like `error_chain` maybe?)